src/main/java/nl/ictm2a4/javagame/services/levels/Level.java

```diff
@@ -0,0 +1,19 @@
+package nl.ictm2a4.javagame.services.levels;
```

Comment:
En zoals in deze bedoelde ik met camelcasing in variabele namen. ID -> iD, Name -> name